src/tgftools/filehandler.py

```diff
@@ -119,7 +119,7 @@ def _checks(_df: pd.DataFrame):
             "indicator",
         ] == list(_df.index.names)
         assert {"low", "central", "high"} == set(_df.columns)
-        assert all_numeric(_df)
+        # assert all_numeric(_df, skipna=True)  # na's in some places is OK
```

Comment:
Why are ok with non-numeric data in ModelResults?


(But, if it is ok, then we can remove this line (rather than comment it out))